### DISCUSSION TITLE
Default values for transactions and custom order in transaction information

### DISCUSSION THREAD

**Discussion by JSchimmelpfennig** [firefly-iii/firefly-iii#9136] [Firefly III]
Hello guys :)
I really love firefly and I would like to make the use more efficient.

I have three source accounts, but I usually just use one. Is there a way to configure predefines values for the files of the transaction summary?

Furthermore, I jump to the fields by using the tab key. The 4th field is the date and it requires 6x pressing tab before I can jump to the amount. Is there a way of customizing the order of the fields in the transaction information?

Thanks in advance :)

**Comment by JC5** [comment]
Hey, great questions!

I'm afraid it's not possible to do defaults, but you could use rules to fill them in after the fact. Some users use "shortcodes" when creating a transaction, to have to fill in as little as possible. Try:

```
https://example.com/transactions/create/withdrawal?source=1
```

Where 1 is your source asset account ID. This will fill in the account. You can then fill the rest, or use a rule to expand the transaction.

The tab order isn't something I can fix unfortunately.
